docs/source/conf.py

```diff
@@ -319,8 +319,9 @@
     ('cpp:identifier', 'sparse_indexing::zero_based'),
     ('cpp:identifier', 'sparse_indexing::one_based'),
     # array
-    ('cpp:identifier', 'array'),
     ('cpp:identifier', 'T'),
+    ('cpp:identifier', 'array'),
+    ('cpp.identifier', 'impl_t'),
```

Comment:
I think an update here requires update in docs (leads to docbuild error) so either update docs as well or remove this for now